### PR TITLE
Alexandria (v0.2.0) compatibilitiy updates for defi/amms/bonding

### PR DESCRIPTION
**Pull request by devmannic**
* pin to v0.2.0 for `scrypto_statictypes` and updated rev for scrypto-unit.
* API updates to use new ResourceBuilder etc.
* bump package version
* remove "pre-Alexandria" note from the README

### CODE REVIEW

**Comment by cbisaillon** [comment]
You should remove the `name = "out"` from the Cargo.toml

File: defi/amms/bonding/tests/lib.rs [hunk 1]
```diff
@@ -27,7 +27,7 @@ fn setup_fixture<'a, L: Ledger>(env: &mut TestEnv<'a, L>) -> (User, User, Resour
 
     const PACKAGE: &str = "bonding";
 
-    env.publish_package(PACKAGE, include_code!());
+    env.publish_package(PACKAGE, include_code!("out"));
```
Comment (cbisaillon) [inline defi/amms/bonding/tests/lib.rs:30]: Don't forget to rename this when you remove the `name="out"`

Comment (devmannic) [reply defi/amms/bonding/tests/lib.rs]: pushed another commit removing the lib name override and fixing this line to match.

Comment (cbisaillon) [reply defi/amms/bonding/tests/lib.rs]: Thanks !

**Pull request merged**